pycaching/cache.py

```diff
@@ -412,6 +424,7 @@ def load(self):
         else:
             trackable_page_url = None
 
+        log_page_url = root.findAll("div", "CacheDetailNavigation NoPrint")[0].find("a", {'id' : 'ctl00_ContentBody_GeoNav_logButton'})["href"]
```

Comment:
Could you select this element by using its ID? The ID should be unique in the whole page, so the selector could be shortened to: `root.find(id="ctl00_ContentBody_GeoNav_logButton")["href"]`